XamarinCommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -160,7 +160,7 @@
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.4" />
   </ItemGroup>
 
-  <ItemGroup Condition=" $(TargetFramework.StartsWith('netcoreapp'))  ">
+  <ItemGroup Condition=" $(TargetFramework.StartsWith('netcoreapp')) AND '$(OS)' == 'Windows_NT' ">
```

Comment:
@jfversluis that's needed for those guys who work on Mac (like myself), otherwise, the app fails to compile